tools/azure-sdk-tools/packaging_tools/templates/README.md

```diff
@@ -14,20 +13,18 @@ For the older Azure Service Management (ASM) libraries, see
 For a more complete set of Azure libraries, see the `azure <https://pypi.python.org/pypi/azure>`__ bundle package.
 
 
-Usage
-=====
+# Usage
```

Comment:
Let's remove `For a more complete set of Azure libraries, see the `azure <https://pypi.python.org/pypi/azure>`__ bundle package.`

Replace it with something like "For a more complete view of the et of Azure libraries, see the Github repo <link to repo>`